onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1120,11 +1121,11 @@ TensorrtExecutionProvider::GetCapability(const GraphViewer& graph,
     if (!group.first.empty()) {
       std::unique_ptr<IndexedSubGraph> sub_graph = GetSubGraph(group, graph);
       result.push_back(ComputeCapability::Create(std::move(sub_graph)));
-      number_of_trt_nodes += group.first.size();
+      number_of_trt_nodes += static_cast<int>(group.first.size());
     }
   }
 
-  const int number_of_subgraphs = supported_nodes_vector.size();
+  const int number_of_subgraphs = static_cast<int>(supported_nodes_vector.size());
```

Comment:
does number_of_subgraphs need to be casted?  it's only used in some logging output.
also can you fix the typos in line 1130, 1132, 1134.  execution is misspelled.

Reply:
Changed to use size_t and also fixed typo.